backend/routes/quests.go

```diff
@@ -75,11 +92,6 @@ type QuestsConfig struct {
 	} `json:"main"`
 }
 
-type QuestStatus struct {
```

Comment:
There is a bunch of stuff removed in this PR that should not have been removed, like this.

Reply:
can you explain further, what was remove that should not be remove

Reply:
i see, i will fix this